Makefile

```diff
@@ -7,7 +7,7 @@ rebuild: clean build
 bin/recall:
 	mkdir -p bin
 	go mod tidy
-	go build -o bin/recall ./cmd/recall
+	go build -tags tui -o bin/recall ./cmd/recall
```

Comment:
Is it possible to make including the tui a default? So we have to do

```
go build --tags notui -o bin/recall ./cmd/recall
```

if we don't want the TUI?

Reply:
I ll investigate

Reply:
Done